dns01/var/named/master/internal/stackato.com.zone

```diff
@@ -280,3 +280,190 @@ vstest1				IN	A	192.168.68.131
 wrong-cn-no-bundle-ssltest	IN	CNAME	ssltest
 wrong-cn-ssltest		IN	CNAME	ssltest
 www				IN	CNAME	stackato.com.
+
+dhcp-68-30			IN	A	192.168.68.30
```

Comment:
I've opted to put this at the bottom of the file because nobody wants to read it. Essentially, both here and alpha-sorted are terrible choices.